README.md

```diff
@@ -150,9 +150,11 @@ Grab the values for the `AWS_ACCESS_KEY_ID` and the `AWS_SECRET_ACCESS_KEY` and
 Now you should be able to start uploading some files!
 
 ## Review Apps
-This repo is set up to deploy review apps on Heroku, and those pull from the staging database to match the experience of deploying as closely as possible! However, note that in order to prevent unapproved model changes from effecting the staging database, migrations are prevented from running on review apps. So those will still have to be reviewed locally.
+This repo is set up to deploy review apps on Heroku, and those pull from the staging database to match the experience of deploying as closely as possible! However, note that in order to prevent unapproved model changes from effecting the staging database, migrations are prevented from running on review apps by default.
 
-### Review apps for Wagtail changes
+If a branch involves a migration, either review those changes locally, or check out the following instructions to provision and populate a database for review apps.
```

Comment:
After doing this a few times, it seems that the instructions below would work fine for more than just wagtail changes and we wouldn't have to review any migrations locally if we don't want to. So I've adjusted the language to make that clear. Let me know if this isn't the case!